packages/react-native-tab-view/src/__tests__/PagerViewAdapter.test.tsx

```diff
@@ -0,0 +1,85 @@
+import { expect, jest, test } from '@jest/globals';
```

Comment:
This test isn't necessary. The tests for the public API should already use appropriate behavior. So remove this.